remote.go

```diff
@@ -1124,6 +1134,35 @@ func (wd *remoteWD) Screenshot() ([]byte, error) {
 	return ioutil.ReadAll(decoder)
 }
 
+func (wd *remoteWD) WaitWithTimeoutAndInterval(condition Condition, timeout time.Duration, interval time.Duration) (error) {
+	startTime := time.Now()
+
+	for {
+		done, err := condition(wd)
+		if err != nil {
+			return err
+		}
+
```

Comment:
nit: remove this empty line, since the two conditions are related to the function call.